src/api/generate-xlsx/map-application-to-XLSX/index.ts

```diff
@@ -21,10 +20,6 @@ const mapApplicationToXLSX = (application: Application) => {
 
       ROW_SEPERATOR,
 
-      ...mapExporterContactDetails(application),
```

Comment:
Same here, this is updated in [this PR](https://github.com/UK-Export-Finance/exip/pull/1078)